pkg/imagestore/imagestore.go

```diff
@@ -132,19 +156,19 @@ func (s *rhcosStore) Populate(ctx context.Context) error {
 					return fmt.Errorf("failed to download %s: %v", url, err)
 				}
 
-				log.Infof("Finished downloading for %s-%s", openshiftVersion, arch)
+				log.Infof("Finished downloading for %s-%s", imageVersion, arch)
```

Comment:
I don't know, each of the "actual" version numbers starts with the ocp major/minor so I don't think it will be too confusing.

That said, it's just a log. I don't have strong opinions on it.